3684/0/about_scoring_project.rb

```diff
@@ -28,43 +28,17 @@
 # More scoring examples are given in the tests below:
 #
 # Your goal is to write the score method.
-def set_points(dice_array, index, multipler)
-  counter = 0
-  dice_array.each do |item|
-    counter += 1 if item == index
-    if counter == 3
-      dice_array.slice!(dice_array.index(index)..dice_array.index(index) + counter - 1)
-      return index * multipler
-    end
-  end
-  0
-end
-
-def solo_points(dice_array, index, multipler)
-  dice_array.each do |item|
-    next unless item == index
-
-    count = dice_array.count(index)
-    dice_array.slice!(dice_array.index(index))
-    return multipler * count
-  end
-  0
-end
 
 def score(dice)
-  dice.sort!
-  points = 0
-
-  # Finding set of three numbers and deleting them
-  points += set_points(dice, 1, 1000)
-  (2..6).each do |item|
-    points += set_points(dice, item, 100)
+  score = 0
+  (1..6).each do |item|
+    amount = dice.count(item)
+    score += (item == 1 ? 1000 : item * 100) if amount >= 3
+    score += (amount % 3) * 100 if item == 1
+    score += (amount % 3) * 50 if item == 5
   end
-  # Finding solo one or five
-  points += solo_points(dice, 1, 100)
-  points += solo_points(dice, 5, 50)
 
-  points
+  score
 end
```

Comment:
please, at least pretend that this is yous...

Reply:
I found similar function on the internet, because mine was too big and difficult and cannot pass rubocop metrics, i did this before i know about disabling metrics

Reply:
delete this file. create the same one and paste a valid code there. I don't wont to see 3501's code as base of your PR. 

Please understand! If Grand Master Tolik gonna see this mess - we are dead! Both!

Reply:
But this function has nothing similar to 3501, just 1 variable, named points, and i think no one will name this like o4ki or koli4estvo_o4kov